src/main/java/nextstep/lotto/utils/ValidationUtils.java

```diff
@@ -0,0 +1,21 @@
+package nextstep.lotto.utils;
+
+public class ValidationUtils {
+
+    private static final String EMPTY_STRING = "";
+
+    private ValidationUtils() {
+    }
+
+    public static boolean isNullAndEmpty(String inputValue){
+        if(inputValue == null){
+            return true;
+        }
+
+        if(inputValue.trim().equals(EMPTY_STRING)) {
+            return true;
+        }
+        return false;
```

Comment:
```suggestion
        if(inputValue == null){
            return true;
        }

        return inputValue.trim().equals(EMPTY_STRING);
```

불필요한 if 구문을 하나 제거할 수 있을 것 같네요~

Reply:
정말 불필요한 if이였네요....
이부분 수정하여 다시 커밋하도록하겠습니다.